usage-examples/build-avoidance.svg

```diff
@@ -0,0 +1,2 @@
#a64d79" stroke-width="2" stroke-miterlimit="10" pointer-events="none"/><path d="M 192.24 362 L 200.24 358 L 200.24 366 Z" fill="#a64d79" stroke="#a64d79" stroke-width="2" stroke-miterlimit="10" pointer-events="none"/><g transform="translate(588.5,389.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="93" height="15" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 94px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;"><div><span style="font-size: 14px ; font-family: &quot;arial&quot; ; white-space: pre-wrap ; text-decoration: none ; line-height: 16.5px ; color: rgb(0 , 0 , 0)">ArtifactReused</span></div></div></div></foreignObject><text x="47" y="14" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><g transform="translate(658.5,459.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="93" height="15" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 94px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;"><div><span style="font-size: 14px ; font-family: &quot;arial&quot; ; white-space: pre-wrap ; text-decoration: none ; line-height: 16.5px ; color: rgb(0 , 0 , 0)">ArtifactReused</span></div></div></div></foreignObject><text x="47" y="14" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><g transform="translate(728.5,519.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="93" height="15" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 94px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;"><div><span style="font-size: 14px ; font-family: &quot;arial&quot; ; white-space: pre-wrap ; text-decoration: none ; line-height: 16.5px ; color: rgb(0 , 0 , 0)">ArtifactReused</span></div></div></div></foreignObject><text x="47" y="14" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica">[Not supported by viewer]</text></switch></g></g></svg>
```

Comment:
I guess you removed the prefixes of the event names for brevity and that might be fine, but the text explaining the graph must then also be updated accordingly. That is true not only for this graph but for all.

Reply:
Could you elaborate? The prefixes and suffixes haven't been in the diagrams for ages (if ever?). I assume you're referring to Eiffel...Event?

Reply:
In the text on build-avoidance the events are referred to as 'CDefC1' and 'ArtCC1' for example. The same names appear in the circles in the current diagrams. In this PR you've remove the event name shortcut prefixes leaving only 'C1' and 'C1' which is not very distinct and clear.